src/cache.js

```diff
@@ -3,13 +3,20 @@ import hash from "object-hash";
 
 export const withCacheAsync = (f, options) => {
   const cache = new NodeCache(options);
-  return async (...args) => {
-    const cacheKey = hash.sha1(args);
-    if (cache.has(cacheKey)) {
-      return cache.get(cacheKey);
-    }
-    const result = await f.apply(this, args);
-    cache.set(cacheKey, result);
-    return result;
-  };
+  return withCacheAsyncCustom(
+    cache.get,
+    cache.set,
+    f
+  )
+};
+
+export const withCacheAsyncCustom = (get, set, f) => async (...args) => {
+  const cacheKey = hash.sha1(args);
+  const value = await get(cacheKey)
+  if (value){
```

Comment:
I would explicitly check if this is null / undefined (what ever you think) as `0` might be a valid response right ? 
and then fix cache.get to transform this into undefined

Reply:
changed to ramda's isNil